vtrace/platforms/win32.py

```diff
@@ -1376,7 +1381,10 @@ def platformDetach(self):
             self.platformWait()
         if not kernel32.DebugActiveProcessStop(self.pid):
             raiseWin32Error("DebugActiveProcessStop")
-        kernel32.CloseHandle(self.phandle)
+        try:
+            kernel32.CloseHandle(self.phandle)
+        except:
+            pass
```

Comment:
given that there are "raises" above, then it seems like there's a handle leak here. perhaps it would be convenient to implement a contextmanager that closes the handle RAII-style?

```py
@contextlib.contextmanager
def closing(handle):
    try:
        yield
    finally:
        kernel32.CloseHandle(handle)

with closing(self.phandle):
    ...